client/src/app/page.tsx

```diff
@@ -1,4 +1,78 @@
-import { CounterText } from "@/ui/CounterText";
+"use client";
+//import { LandingNavbar } from "@/ui/LandingNavbar";
+import "./LandingPage.css";
+import Image from "next/image";
+//import { TextCarousel } from "@/ui/TextCarousel";
+import { Typography } from "@mui/material";
+import Link from "@mui/material/Link";
+
 export default function Home() {
-  return <CounterText />;
+  {
+    /*Added Image Array to allow for easy image changes*/
+  }
+  const ImageData = [
+    {
+      // Section 1 Logo Image
+      imageSrc: "/LandingPage-images/rat_1.png",
+      imageAlt: "SqueakPeek Logo",
+      imageWidth: 300,
+      imageHeight: 300,
+    },
+    {
+      // Section 2 Threads Feature
+      imageSrc: "/LandingPage-images/rat_2.png",
+      imageAlt: "Two rats speaking",
+      imageWidth: 600,
+      imageHeight: 300,
+    },
+    {
+      // Section 3 Tracking Feature
+      imageSrc: "/LandingPage-images/rat_1.png",
+      imageAlt: "Rat looking up",
+      imageWidth: 400,
+      imageHeight: 400,
+    },
+    {
+      // Section 4 Explore Feature
+      imageSrc: "/LandingPage-images/rat_3.png",
+      imageAlt: "Rat looking at star",
+      imageWidth: 400,
+      imageHeight: 400,
+    },
+  ];
+
+  return (
+    <div>
+      {/* <LandingNavbar /> Cant be referrenced yet*/}
+
+      {/*Section 1 Block*/}
+      <div className="sec1-container">
```

Comment:
I would make this abstract this UI into it's own component

Reply:
Dont believe this is necessary

Reply:
This would make it easier to make changes in the future by having it live in it's own component. I would highly recommend make it into its UI component so all relevant parts of it is under one file.